index.html

```diff
@@ -15,8 +15,8 @@ <h1>Task Manager</h1>
             <input type="text" id="task" placeholder="Add a task" />
             <button type="submit" id="addBtn">Add</button>
         </form>
-        <div class="buttons">
-            <button id="viewBtn">View Tasks</button>
+        <div id="tasksCnt">
+            Count tasks: 0
         </div>

```

Comment:
@koteniatko,
suggestion: It's better to create separate tag for number. Text should remain in HTML.

Ex.:
```
<div>
      Count tasks: <span  id="tasksCnt">0</span>
  </div>
```

So that in JS you can specify only number, like: `document.getElementById('tasksCnt').innerText = '5'`